src/bitmessagekivy/baseclass/addressbook_widgets.py

```diff
@@ -10,9 +10,12 @@
 
 import state
 
+no_address_found = "No contact found yet......"
+empty_search_label = "No contact found!"
 
-class HelperAddressBook(object):
-    """Widget used in Addressbook are here"""
+
+class DefaultLabelMixin(object):
```

Comment:
if it's a mixin, no `object`. You may have to override pylint, as it's currently using python2